nrf_wifi/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -338,6 +338,17 @@ enum MAX_POWER_OFFSETS {
 	NRF_WIFI_MAX_OP_PWR_5GHZ_HI_MCS0
 };
 
+/** The byte offsets of RF parameters indicate the start offset
+ * of PCB loss for 2.4 GHz and 5 GHz bands.
+ */
+enum PCB_LOSS_BYTE_OFFSETS {
```

Comment:
If these are location offsets in the PKTRAM shouldn't they be part of `rpu_if.h` and be `#defines` instead of `enum` ?

Reply:
`The byte offsets of RF parameters`

Reply:
Followed the pattern that's already been used for other RF parameter offsets.